### ISSUE TITLE
[Design, FE] My VA 508 office audit: Review and resolve headings and labels findings

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#78974]
## Background
There was one low level defect (that our team can address) found in the audit related to headings and labels.

**WCAG 2.4.6 Headings and Labels | Headings and labels describe topic or purpose.**
Log in with user `vets.gov.user+1@gmail.com` and go the main my-va page. There are hidden sr-only headings that are confusing. For example Claims and Appeals is an H2 tag and is visually and audially marked. The next Header tag when using the screen reader is "Popular Actions for Claims and Appeals" which is a not a visual header tag. The hidden header tag serves no purpose. This happens elsewhere on the page as well.

Additionally, the Benefits Payments section has the +836.13 as a header. Headers are meant to break the page into sections and describe the topic or purpose of that section. This header does not meet that requirement. It should just be text. 

## Tasks
- [ ] Review defect details
- [ ] Implement fix 
- [ ] Test with Matt or other designer 

## Acceptance Criteria
- [ ] Heading and label finding is resolved

**Comment by DanielMiller-agile6** [comment]
verified on staging that these changes are deployed and working as intended

**Comment by DanielMiller-agile6** [comment]
also, I'm not sure if there is a 508 test verification process or anything. will the tester(s) have a chance to retest with these changes? if so, we can reopen this if needed until they retest
